xbmc/interfaces/python/swig.cpp

```diff
@@ -377,5 +377,8 @@ namespace PythonBindings
     return typeInfoLookup[ti];
   }
 
+  int dummy_tp_init(PyObject *self, PyObject *args, PyObject *kwds) {
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types

> Curly braces always go on a new line.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#31-braces

Reply:
fixed